sds/src/standard_rules/mod.rs

```diff
@@ -0,0 +1,290 @@
+use crate::{Labels, MatchAction, RegexRuleConfig, Scope};
+use crate::{ProximityKeywordsConfig, RuleConfigTrait, SecondaryValidator};
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::fs::File;
+use std::io::BufReader;
+use std::path::Path;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct StandardRule {
+    pub id: String,
+    pub name: String,
+    pub description: String,
+    pub pattern: String,
```

Comment:
@fuchsnj can you add a type there 

```
                if (rule.getScannerType().equals(ScannerType.CREDIT_CARD_SCANNER)) {
                  builder.secondaryValidator(SecondaryValidator.LUHN_CHECKSUM);
                } else if (rule.getScannerType().equals(ScannerType.CHINESE_ID_SCANNER)) {
                  builder.secondaryValidator(SecondaryValidator.CHINESE_ID_CHECKSUM);
                } else if (rule.getScannerType().equals(ScannerType.NAME_SCANNER)) {
                  builder.type(SdsRule.Builder.NAME_DETECTION_TYPE);
```

Reply:
And a bool experimental

Reply:
I added an experimental bool field. As for the type, I'd like to add that in a separate PR. I believe all existing standard rules (unless I'm missing some) are Regex today. Adding other types will likely end up changing the structure of the rules a bit (e.g. `pattern` is no longer required). The set of changes to get the existing rules migrated is already large enough, I'd like to keep other improvement separate.

Reply:
Ok for an other PR FYI I am current working on a branch that have type = name_detection ;)